code/modules/medical/computer/cloning.dm

```diff
@@ -447,12 +447,21 @@
 			break
 	return selected_record
 
+/obj/machinery/computer/cloning/process()
+	update_icon()
```

Comment:
it's not that expensive, it needs to update whenever someone is moved in/out of a machine, and in the case of the operating computer, every time someone is moved on/out of the operating table.